src/app/workflow/launch/launch.component.html

```diff
@@ -82,6 +82,31 @@
         </mat-card>
         <app-launch-checker-workflow [versionName]="_selectedVersion?.name" [command]="checkEntryCommand"></app-launch-checker-workflow>
       </div>
+
+      <mat-card *ngIf="(descriptorType$ | async) === ToolDescriptor.TypeEnum.WDL && (published$ | async)">
+        <div [matTooltip]="wesTooltip">
+          <p>Launch this workflow on Amazon Web Services' (AWS) Amazon Genomics CLI (AGC) cloud infrastructure.</p>
+
+          <!--TODO: replace with actual documentation link after https://github.com/dockstore/dockstore-documentation/pull/173-->
```

Comment:
Do you know what the url will be? If so, I would put in now, even if it doesn't exist yet, then we don't have to remember to change this code again before the release.